src/Core/Package.vala

```diff
@@ -386,6 +388,37 @@ public class AppCenterCore.Package : Object {
         return latest_version;
     }
 
+    public string? get_author () {
+        if (author != null) {
+            return author;
+        }
+
+        author = component.developer_name;
+
+        if (author == null) {
+            var project_group = component.project_group;
+
+            if (project_group != null) {
+                author = project_group;
+            }
+        }
+
+        return author;
+    }
+
+    public string get_author_title () {
```

Comment:
Can we use properties instead of `get` methods please? It's more vala style to have `package.author_title`